src/qcodes_contrib_drivers/drivers/SPECS/Specs_Tramea.py

```diff
@@ -0,0 +1,1524 @@
+from typing import Optional
+
+from qcodes import Instrument
+import socket, nanonis_tramea_specs
+import numpy as np
+"""
+IMPORTANT
+
+Before using the Driver, you must have our python package installed, because many of functions in this driver are merely
+wrapper functions of functions from said package. How to install:
+
+pip install nanonis-tramea-specs
+"""
+
+
+class NanonisTramea(Instrument):
+    # log = logging.getLogger(__name__)
+
+    def __init__(self, name: str, address: str, port: int, **kwargs):
+        super().__init__(name, **kwargs)
+
+        self._address = address
+        self._port = port
+
+        self.model = "SPM"
+        self.serial = 1234
+        self.firmware = 1
+
+        self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self._socket.connect((self._address, self._port))
+
+        self.n = nanonis_tramea_specs.Nanonis(self._socket)
+        # self.n.Bias_Set(np.float32(1.2))
+
+        self.add_parameter(
+            "Output1",
+            set_cmd=self.UserOut_ValSet1,
+            get_cmd=self.Signals_ValGet24,
```

Comment:
Can we directly use `self.n` attributes instead of defining individual functions for each channel?
```python
    set_cmd=partial(self.n.UserOut_ValSet, 1)
    get_cmd=(self.n.Signals_ValGet(24, 1)[2])[0]
```

To verify, is it intended that `get_cmd=self.Signals_ValGet24` (specifically 24) for all `OutputX` parameters?

Reply:
I was unaware that you could write partial functions like this in python (this is my first project in python). This is clearly way better. And no, that is not intended, I must have accidentally done that. I will change this immediately. Thank you for the input!